eng/Versions.props

```diff
@@ -65,7 +65,7 @@
     <MicrosoftInternalVisualStudioShellInterop140DesignTimeVersion>14.3.25407-alpha</MicrosoftInternalVisualStudioShellInterop140DesignTimeVersion>
     <MicrosoftMetadataVisualizerVersion>1.0.0-beta1-63011-01</MicrosoftMetadataVisualizerVersion>
     <MicrosoftMSXMLVersion>8.0.0.0-alpha</MicrosoftMSXMLVersion>
-    <MicrosoftNetCompilersToolsetVersion>3.1.0-beta1-19164-01</MicrosoftNetCompilersToolsetVersion>
+    <MicrosoftNetCompilersToolsetVersion>3.2.0-beta2-19275-07</MicrosoftNetCompilersToolsetVersion>
```

Comment:
This version of the toolset has a workaround for `GetPathsOfAllDirectoriesAbove`

Reply:
Any reason this isn't flowing from Arcade (which might encourage regular updates/making those updates automated)?

Reply:
CC. @jaredpar

Reply:
At times we need a newer version than what Arcade offers just do to bootstrapping. In general we should be flowing from Arcade but there are specific points where we need to override it. 

---
In reply to: [289079090](https://github.com/dotnet/roslyn/pull/36042#discussion_r289079090) [](ancestors = 289079090)